README.md

```diff
@@ -0,0 +1,106 @@
+# GroupDocs.Conversion for .NET MVC Example
+###### version 0.1.0
+
+[![Build status](https://ci.appveyor.com/api/projects/status/boo4pnp61r6b8kqp/branch/master?svg=true)](https://ci.appveyor.com/project/egorovpavel/groupdocs-conversion-for-net-mvc/branch/master)
```

Comment:
[![Build status](https://ci.appveyor.com/api/projects/status/dl03chqh21wwkysj/branch/master?svg=true)](https://ci.appveyor.com/project/egorovpavel/groupdocs-conversion-for-net-mvc/branch/master)